ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -1310,6 +1310,11 @@ let gen_cmds rpc session_id =
         mk get_all_records_where get_by_uuid vtpm_record "vtpm" []
           ["uuid"; "vm"; "profile"] rpc session_id
       )
+    ; Client.Tracing.(
+        mk get_all_records_where get_by_uuid tracing_record "tracing" []
```

Comment:
Same comment, the API calls it tracer provider, we want to expose only a single name to avoid confusing users

Reply:
That is a the class name, I remever there being some discussion on the name of the class